### ISSUE TITLE
Crash Dino 0.4

### ISSUE THREAD

**Issue by keshaTut** [dino/dino#1366]
build from source
launching Dino 0.4 build/dino
Everything is loaded, everything works
When the pop-up window is closed, the Dino crashes. These windows include application settings, chat settings, and reaction setting. However, if you open and close the emoji list for the chat itself (click on the smiley), it does not crash. After the crash, when I go to Dino again, I see the reaction set. If I decide to remove this reaction (click on the smiley under the message), then Dino will crash again.
gtk-doc 1.33.2-1
gtk-update-icon-cache 1:4.8.3-3
gtk2 2.24.33.2
gtk3 1:3.24.36-1
gtk4 1:4.8.3-3
gtksorceview4 4.8.4-1
kde-gtk-config 5.26.5-1

https://pastebin.com/E2wQrS0U

**Comment by licaon-kter** [comment]
Support channel info said GTK4 4.9.1 fixes it, or older 4.6.

**Comment by fiaxh** [comment]
No, I actually asked them to file the issue, since it sounds different than the GTK thing.
